extensions/twenty/src/components/MultiSelect.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { forwardRef } from "react";
+import { Form, FormItemRef } from "@raycast/api";
+import { DataModelField } from "../services/zod/schema/recordFieldSchema";
+import { optionIcons } from "../enum/icons";
+
+type MultiSelectProps = {
+  field: DataModelField;
+};
+
+// Use forwardRef for consistency and ref handling
+const MultiSelect = forwardRef<FormItemRef, { values: MultiSelectProps } & React.ComponentProps<typeof Form.TagPicker>>(
+  ({ values, ...rest }, ref) => {
+    const { field } = values;
+    const { options } = field;
+
+    const defaultValue = Array.isArray(field.defaultValue)
+      ? field.defaultValue.map((value) => value.replace(/^'|'$/g, "").trim())
+      : field.defaultValue
+        ? [field.defaultValue.replace(/^'|'$/g, "").trim()]
+        : [];
+
+    const { id, value, ...modifiedRest } = rest; // eslint-disable-line @typescript-eslint/no-unused-vars
+
+    return (
+      <Form.TagPicker
+        title={field.label}
+        defaultValue={defaultValue}
+        id={field.name}
+        ref={ref as React.Ref<FormItemRef>}
```

Comment:
@GeniusNirmit Here add placeholder `Select ${field.name}...`.  see TextInput Field for reference.